stable_baselines3/common/distributions.py

```diff
@@ -325,23 +323,23 @@ def proba_distribution_net(self, latent_dim: int) -> nn.Module:
         return action_logits
 
     def proba_distribution(self, action_logits: th.Tensor) -> "MultiCategoricalDistribution":
-        self.distributions = [Categorical(logits=split) for split in th.split(action_logits, tuple(self.action_dims), dim=1)]
```

Comment:
why do you need the rename here? (as you are using a different implementation below)

Reply:
Mainly for conformity, there's no real upside to having it be named differently than all the other distributions I'd argue it just makes things confusing when using it, especially since `self.distribution` would also be callable since it's in the base class now.